site/content/docs/5.3/utilities/opacity.md

```diff
@@ -9,7 +9,43 @@ aliases:
   - "/docs/utilities/opacity/"
 ---
 
-The `opacity` property sets the opacity level for an element. The opacity level describes the transparency level, where `1` is not transparent at all, `.5` is 50% visible, and `0` is completely transparent.
+The `opacity` property sets the opacity level for an element.
+
+The opacity level describes the semantic transparency level, where `opaque` is not transparent at all, `emphasis` to `weaker` are various levels of transparency, and `transparent` is completely transparent.
+
+Set the opacity of an element using `.opacity-{value}` utilities.
+
+<!--TODO-->
+
+<div class="bd-example d-sm-flex">
+  <div class="opacity-opaque p-3 m-2 bg-primary fw-bold"></div>
+  <div class="opacity-emphasis p-3 m-2 bg-primary fw-bold"></div>
+  <div class="opacity-medium p-3 m-2 bg-primary fw-bold"></div>
+  <div class="opacity-weak p-3 m-2 bg-primary fw-bold"></div>
+  <div class="opacity-weaker p-3 m-2 bg-primary fw-bold"></div>
+  <div class="opacity-transparent p-3 m-2 bg-primary fw-bold"></div>
+</div>
+
+```html
+<div class="opacity-opaque">...</div>
+<div class="opacity-emphasis">...</div>
+<div class="opacity-medium">...</div>
+<div class="opacity-weak">...</div>
+<div class="opacity-weaker">...</div>
+<div class="opacity-transparent">...</div>
+```
+
+<details style="border: 1px dashed black" class="p-2 mb-4">
```

Comment:
The rendering must be improved, use class, and maybe use shortcodes as well. It's just for now a suggestion of rendering.